hlxsites/merative2
# Issue 954 / 955

## Issue

Fixes https://merative.atlassian.net/browse/MERATIVE-954
Fix : https://merative.atlassian.net/browse/MERATIVE-955

## Description
https://merative.atlassian.net/browse/MERATIVE-955
> When a content author adds the references block to a blog post, the Unordered lists/numbered lists and list items are not sized correctly. When the references block is removed, this issue gets fixed.

https://merative.atlassian.net/browse/MERATIVE-954
> Tertriary button in leadspaces should be center aligned in the leadspace block. Currenlty it is aligned bottom.

## Design Specs

> If applicable, add the direct link to the design specs of the component/feature that's part of this PR.

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=10089%3A44814&mode=dev&t=lwqWwvMQKCZdofLG-1

## Test URLs
  
Fixes https://merative.atlassian.net/browse/MERATIVE-954
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/clinical-decision-support
- After (Changes from this PR): https://issue-914--merative2--hlxsites.hlx.page/clinical-decision-support

Fix : https://merative.atlassian.net/browse/MERATIVE-955
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/block-library/templates/blog
- After (Changes from this PR): https://issue-914--merative2--hlxsites.hlx.page/block-library/templates/blog


  
## Testing Instruction

> Open each New URL and inspect the code changes to ensure they render according to the Design Specs in Figma



Note:
The changes are generic and affects all the support pages where "columns" and "leadspace" blocks are used.
